data/tribes/workers/frisians/reed_farmer/init.lua

```diff
@@ -18,17 +18,17 @@ tribes:new_worker_type {
       plantreed = {
          "findspace=size:any radius:1",
          "walk=coords",
-         "animate=dig 2000",
-         "animate=planting 1000",
+         "animate=dig duration:2s",
+         "animate=planting duration:1s",
          "plant=attrib:seed_reed",
          "return"
       },
       harvestreed = {
          "findobject=attrib:ripe_reed radius:1",
          "walk=object",
-         "animate=harvesting 12000",
+         "animate=harvesting duration:12s",
          "callobject=harvest",
-         "animate=harvesting 1",
+         "animate=harvesting duration:1ms",
```

Comment:
that might be a bug we should consult @Noordfrees about this. (1ms would be barely visible)

Reply:
Indeed this should best be twice `6s` rather than `12s` + `1ms` 😁

Reply:
Done.

I have run the test suite locally and will merge now to give our CI a break.